run.sh

```diff
@@ -0,0 +1,19 @@
+
+set -e
+
+source /opt/adapteva/esdk/setup.sh
+
+if [ ! -f /app/installed.txt ]; then
+	echo "Building FFT correlation"
+	cd /app/parallella-fft-xcorr && PATH=/usr/local/browndeer/bin:$PATH LD_LIBRARY_PATH=/usr/local/browndeer/lib:$LD_LIBRARY_PATH make IMPL=coprthr
+	install -m 644 /app/parallella-fft-xcorr/libfft-demo-coprthr.so /usr/lib/
+	ldconfig /usr/lib /usr/local/browndeer/lib
+
+	cp /app/parallella-fft-xcorr/device.cbin.3.e32 /app/
+	echo "Installing app"
+	cd /app && npm install --unsafe-perm
```

Comment:
Not really. I got to this after fighting a while with node-gyp and incompatible v8 versions. But I'll do the npm install at build time and just run `node-gyp configure build here`.